### PR TITLE
Reorder Suppression fixes to be above Configuration fixes in the lightbulb

### PR DESCRIPTION
**Pull request by dpoeschl**
Fixes https://github.com/dotnet/roslyn/issues/38914

image

### CODE REVIEW

**Comment by dpoeschl** [comment]
@sharwell @CyrusNajmabadi @mavasani Does the screenshot look how you expected it?

**Comment by dpoeschl** [comment]
Test failures are legitimate. Will fix.

**Comment by mavasani** [comment]
Fix LGTM, but we have integration tests for configuration fixes, which should break with this change. Are those disabled?

**Comment by mavasani** [comment]
Ah, seems the integration test was skipped while we still had legacy editorconfig document option provider: https://github.com/dotnet/roslyn/blob/a146b803df39268f65bb07a06482094e3616b00a/src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs#L647-L697

We may want to try and re-enable the integration test. I can do it as a follow-up PR once this one goes in.

**Comment by dpoeschl** [comment]
@tmeschter Are these string changes okay at this point in time?
@jinujoseph Do I need any additional approval (assuming the rest of the tests pass)?

**Comment by jinujoseph** [comment]
@dpoeschl, feel free to merge when green

**Comment by dpoeschl** [comment]
Test failures seem unrelated, filed as https://github.com/dotnet/roslyn/issues/39696

File: src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx [hunk 1]
```diff
@@ -153,7 +153,7 @@
   <data name="Regex_OtherEscape" xml:space="preserve">
     <value>Regex - Other Escape</value>
   </data>
-  <data name="Configure_or_Suppress_issues" xml:space="preserve">
-    <value>Configure or Suppress issues</value>
+  <data name="Suppress_or_Configure_issues" xml:space="preserve">
+    <value>Suppress or Configure issues</value>
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx]: I'll admit it's not clear to me why "Configure" is also capitalized here...

Comment (jasonmalinowski) [reply src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx]: (it just always looked funny to me...)

**Pull request merged**